src/seedu/addressbook/Main.java

```diff
@@ -31,12 +33,12 @@
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
 
-    public static void main(String... launchArgs) {
+    public static void main(String... launchArgs) throws StorageFileDeletedException {
```

Comment:
this should not be the case. if main throws, then the program halts, right? who will catch this exception?

please recollect the discussion around this during the tutorial and try to make amends going further. no need to resubmit this PR this time.